docs/sriov.md

```diff
@@ -1,4 +1,66 @@
-# Enable SR-IOV on the host
+# Architecture
+
+There are multiple components involved to provide SR-IOV support in KubeVirt
+environment.
+
+* [SR-IOV device plugin](https://github.com/intel/sriov-network-device-plugin)
+  discovers available SR-IOV resources, advertises them to Kubernetes resource
+  manager, and allocates them as needed to pods and VMIs. This component
+  doesn't modify host resources, only discovers and allocates them. For VMIs,
+  allocates `vfio` device nodes to pods; also sets environment variables with
+  PCI IDs of allocated devices. These variables are later used by KubeVirt to
+  configure libvirt domain.
```

Comment:
I'd explicitly mention that adds the resources to the corresponding node capacity section, which is used together with the requests / limits values mentioned below to perform the scheduling.